scalaparse/test/src-2.13-jvm/scalaparse/ScalacParser.scala

```diff
@@ -0,0 +1,61 @@
+package scalaparse
```

Comment:
@He-Pin is this change intentional?

Reply:
Yes, because the api in nsc is different in Scala 2.12 and newer version of scala 2.13.

Reply:
So I have to separate this test for Scala 2.12 and Scala 2.13, maybe I can split it into a dedicated PR. but as the commits is one by one, maybe that's fine too.

Reply:
related link: https://github.com/scala/scala/pull/10406